python/llm/example/langchain/native_int4/voiceassistant.py

```diff
@@ -30,6 +30,7 @@
 import speech_recognition as sr
 import pyttsx3
```

Comment:
I think we can make it transformer int4 example. Create a whisper folder and put it there. Add README if necessary. Just follow how it is done for other transformer int4 examples.

Reply:
yes, we can add a whisper example